flyway/src/main/java/io/micronaut/flyway/FlywayConfigurationProperties.java

```diff
@@ -53,6 +57,7 @@ public class FlywayConfigurationProperties implements Toggleable {
     private String url;
     private String user;
     private String password;
+    private Map<String, String> properties = new Hashtable<>();
```

Comment:
why `HashTable`? This will be a singleton which will be used only on startup. It does not need to support synchronisation.

Reply:
If we don't want to use `HashMap`, I think we should use https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ConcurrentHashMap.html

Reply:
What's the Base Java version for Micronaut Flyway ?

Reply:
17

Reply:
Can I switch this to `List.of()` instead?

Reply:
I have changed this to HashMap https://github.com/micronaut-projects/micronaut-flyway/pull/450/commits/910c8b2336290e1dd4f23129cc74d8aac555f096